doc/flame/components.md

```diff
@@ -44,6 +34,11 @@ throughout the component's lifetime. This method will only run if the parent is
 If the parent is not mounted yet, then this method will wait in a queue (this will have no effect
 on the rest of the game engine). 
 
+The `respectCamera` variable can be overridden or set to `false` (defaults to `true`) to make the
```

Comment:
Didn't we refactor away this variable? :thinking:

Reply:
Lol, true, this variable is now gone too. I'll remove this paragraph then

Reply:
Very out of the scope for this PR, but maybe you could remove this line mentioning it too?
https://github.com/flame-engine/flame/blob/main/packages/flame/lib/src/game/mixins/game.dart#L202